wandb/apis/public/api.py

```diff
@@ -600,7 +600,9 @@ def _parse_path(self, path):
             parts[-1] = parts[-1].split(":")[0]
         elif parts[-1]:
             id = parts[-1]
-        if len(parts) > 1:
+        if len(parts) == 1 and project != "uncategorized":
```

Comment:
could you leave a comment here... since it requires special "knowledge" of default project name. Also it looks like it would break if we change the default project name?

Reply:
Thanks for the review Katya! So I initially added the condition `if len(parts) == 1` as I thought the only case would be if user is passing only `run_id` but then realized `len(parts) == 1` also if user is passing `project:run_id` as previous lines will remove id from parts as well, so this only happens if the project has been set previously either with env variable or in the `overrides` argument.

it looks like it would break if we change the default project name?
So you mean if we use something different from "uncategorized"?

Reply:
yes regarding the second point

Reply:
I see, I don't see any `default_project` variable so should I just add it to handle this situation?

Reply:
ok looking at the code again it seems that it should be fine, it is hard coded all over this file.